global/css/_global.css

```diff
@@ -204,21 +204,43 @@ svg {
 
 .component-slider-single.component-slider-peek .swiper-slide-prev > *,
 .component-slider-single.component-slider-peek .swiper-slide-next > * {
-  @apply su-opacity-[.27] md:su-opacity-100 su-pointer-events-none md:su-pointer-events-auto;
+  @apply su-opacity-25 md:su-opacity-100 su-pointer-events-none md:su-pointer-events-auto;
+}
+
+.component-slider-single.component-slider-peek.component-slider-vertical-videos .swiper-slide-prev > *,
+.component-slider-single.component-slider-peek.component-slider-vertical-videos .swiper-slide-next > * {
+  @apply su-opacity-25 su-pointer-events-none;
+}
+
+.su-slider-dark .component-slider-single.component-slider-peek.component-slider-vertical-videos .swiper-slide-prev > *,
+.su-slider-dark .component-slider-single.component-slider-peek.component-slider-vertical-videos .swiper-slide-next > * {
+  @apply su-opacity-40 su-pointer-events-none;
```

Comment:
For vertical video panel over background image with overlay, I have the "peek" slides at opacity 40% instead of the regular 25% since when viewed over the background image overlay, it needs to be slightly more opaque to stand out a little bit more from the background.